### DISCUSSION TITLE
Multiple issues on security onion 2.3.2 mode standalone config network dhcp

### DISCUSSION THREAD

**Discussion by Axtrus** [Security-Onion-Solutions/securityonion#1973] [Unsupported Versions] [answered]
Hello team,
by the way, I installed and configured my sec onion with a dhcp as management interface and wanting to change the IP that I had done with the installation
most modules and services no longer work
I had several errors like:
- host 'x.x.x.x' is not allowed to connect to this mysql server
- mysql error 1044: access denied for user 'root' @ '%' to database
- nginx 500 internal server error

Could I have your assistance so that everything works with the dhcp on the management interface?
And tell me how if possible to solve these problems above?

**Comment by dougburks** [comment, marked as answer]
From the installer:
![image](https://user-images.githubusercontent.com/1659467/99552073-9b243880-298a-11eb-86d3-5979754a754f.png)

Also mentioned in the docs:
https://docs.securityonion.net/en/2.3/configuration.html

**Comment by Axtrus** [comment]
Hi Dougs

Does installation and configuration still require an internet connection ?
I would like to do a production installation without internet access can it
work?

Le mer. 18 nov. 2020 à 17:42, Doug Burks <notifications@github.com> a
écrit :

**Reply by dougburks** [reply]
If you don't have Internet access, you can choose the Airgap option:
https://docs.securityonion.net/en/2.3/airgap.html

**Comment by Axtrus** [comment]
So without internet for airgap option
How can I install gnome desktop because so-analyst-install need internet ?


Le mer. 18 nov. 2020 à 18:02, Doug Burks <notifications@github.com> a
écrit :

**Reply by dougburks** [reply]
If you choose the Airgap option, then you won't be able to install GNOME desktop right now. However, you can connect from any other machine with a standard web browser.

If you have further questions or problems, please create a new Discussion rather than replying to this DHCP discussion.
